app/models/patient.rb

```diff
@@ -61,6 +92,13 @@ def current_age
     end
   end
 
+  private
+
+  def low_risk?
+    latest_scheduled_appointment.overdue_for_over_a_year? &&
```

Comment:
Where did this definition come from?

Reply:
Roughly from here: https://github.com/simpledotorg/documentation/blob/master/simple.org%20major%20flows/high%20risk%20label.jpg

I'm now thinking that `low_priority` makes more sense?

Since, there's levels of priority which map onto risk. From "Very high" to "Low". They are computed by a combination of the patient's medical history, blood pressures and appointment information (specifically overdue appointments). The risk label is only shown when the priority is highest, otherwise the distinction is made only for sorting purposes.